src/HotChocolate/OpenApi/src/HotChocolate.OpenApi/Helpers/GraphQLNamingHelper.cs

```diff
@@ -0,0 +1,99 @@
+using System.Text;
+using System.Text.Json;
+using HotChocolate.OpenApi.Extensions;
+using HotChocolate.Types;
+
+namespace HotChocolate.OpenApi.Helpers;
+
+internal static class GraphQLNamingHelper
+{
+    public static string CreateInputTypeName(
```

Comment:
I think we should create a lib that deals with these things. we have this code at least 3 times now .... it does not matter for this PR ... but we should create a core lib dealing with naming and reformatting names etc...